components/inputs/docs/input-date-time.md

```diff
@@ -112,82 +112,110 @@ To make your usage of `d2l-input-date-time` accessible, use the following proper
 
 The `input-date` and `input-time` components do not handle timezone and so require the input to be in the user's timezone, which corresponds to the timezone in the `data-timezone` attribute on the `html` element.
 
-### Convert UTC Date/Time to Local Date
+### input-date: Convert UTC Date/Time to Local Timezone and Back to UTC
 
-To convert a UTC date/time (presumably in ISO 8601 format, `YYYY-MM-DDTHH:mm:ss.sssZ`, adjust if not) to a date in the user's timezone for use in `input-date`:
-- Parse into object (`parseISODateTime` from `helpers/dateTime.js`)
-- Convert UTC object into local timezone object (`convertUTCToLocalDateTime` from the [intl library](https://github.com/BrightspaceUI/intl#datetime-conversion-based-on-user-timezone))
-- Format as ISO 8601 date (`formatDateInISO` from `helpers/dateTime.js`)
+To convert a UTC date/time (presumably in ISO 8601 format, `YYYY-MM-DDTHH:mm:ss.sssZ`, adjust if not) to a date in the user's timezone for use in `input-date`, use `getLocalDateFromUTCDateTime` from `helpers/dateTime.js`, which will return the date in the user's timezone in the ISO 8601 date format `YYYY-MM-DD`.
+
+To convert back to UTC, use initial UTC date/time and `getLocalTimeFromUTCDateTime` to get the local time, then `getUTCDateTimeFromLocalDateTime(date, time)` to convert back to UTC.
 
 ```javascript
-import {convertUTCToLocalDateTime} from '@brightspace-ui/intl/lib/dateTime.js';
-import {formatDateInISO, parseISODateTime} from '@brightspace-ui/core/helpers/dateTime.js';
+import {getLocalDateFromUTCDateTime, getLocalTimeFromUTCDateTime, getUTCDateTimeFromLocalDateTime} from '@brightspace-ui/core/helpers/dateTime.js';
 
 const UTCDateTime = '2018-03-01T12:20:00.000Z';
-const UTCDateTimeObject = parseISODateTime(UTCDateTime);
-const localDateTime = convertUTCToLocalDateTime(UTCDateTimeObject);
-const value = formatDateInISO(localDateTime); // use as value in input-date
+const localTime = getLocalTimeFromUTCDateTime(UTCDateTime); // for later conversion back to UTC
+let value = getLocalDateFromUTCDateTime(UTCDateTime); // use as value in input-date
```

Comment:
This example is interesting, where they have a time and want to preserve it because the date-input will obliterate it. I'm wondering if this actually matters in real life... where they'd have a UTC date with a time, want to throw that into a date input but then hold onto the time even if the date changes? I can't really see anyone wanting that? Seems like we could simplify the examples a lot by just not worrying about this edge case?